index.markdown

```diff
@@ -23,8 +23,19 @@ goals:
   - goal: "Fast, reliable and secure implementation of Python that can be used with Rust or compiled to WebAssembly. "
 ---
 
-# Why RustPython, not CPython
+# Why RustPython?
 
-Edit me in index.markdown file. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+There are many implementations of Python. For example:
+- [CPython](https://github.com/python/cpython) (C), which is the most common
+- [Jython](https://www.jython.org/) (Java)
+- [IronPython](https://ironpython.net/) (.NET)
+- [PyPy](https://www.pypy.org/) (Python)  
+- [Stackless](http://www.stackless.com/)
 
-Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+Each of these implementations offer some benefits: Jython, for example, compiles Python source code to Java byte code, then routes it to the Java Virtual Machine. Because Python code is translated to Java byte code, it looks and feels like a true Java program at runtime and so it integrates well with Java applications.   
+
+IronPython is well-integrated with .NET, which means IronPython can use the .NET framework and Python libraries or vice versa.
+
+We want to unlock the same possibilities that Jython and IronPython enable, but for the Rust programming language. Then we want to do more. Using RustPython, we want to compile Python to WebAssembly and allow users to run their Python code natively in the browser.  
```

Comment:
```suggestion
We want to unlock similar possibilities as Jython and IronPython, but for the Rust programming language. RustPython can be embedded into any program or application as a scripting language, with an easy to use API for exporting Rust types to Python. It can even be used in Rust projects that are compiled to WebAssembly, allowing developers or users to run their Python code easily in the browser.
```